corehq/apps/users/analytics.py

```diff
@@ -82,8 +82,11 @@ def get_search_users_in_domain_es_query(domain, search_string, limit, offset):
     return user_es.start(offset).size(limit).sort('username.exact')
 
 
-def get_role_user_count(domain, role_id):
+def get_role_user_count(domain, role_id, web_users_only=True):
```

Comment:
I'm not in love with this flag, but I was too lazy to split out a hidden function and a separate `get_web_user_role_count` function. Open to pushback.

Reply:
Will the default value of `web_users_only=True` give this function a different default behavior? That is, it looks like a new filter is added to the query by default where it was not before. Was that change intentional?

Reply:
not intentional, thank you - 494546b934d